.github/workflows/dusk_ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           RUSK_PROFILE_PATH: "/var/opt/build-cache"
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features --release -- -D warnings
+          args: --all-features --release -- -D warnings -A clippy::derive_partial_eq_without_eq
```

Comment:
I would prefer if these warning were fixed where appropriate, but it can be done separately.

Reply:
Me too, but I want to preserve the prover keys compatibility for now...